Testscripts/Linux/utils.sh

```diff
@@ -2209,11 +2209,8 @@ function install_sshpass () {
 function add_sles_benchmark_repo () {
 	if [ $DISTRO_NAME == "sles" ]; then
 		case $DISTRO_VERSION in
-			11*)
```

Comment:
I still think we should keep the code lines for the older version. We may run some tests on demand

Reply:
The old URL doesn't exist any more.